### ISSUE TITLE
Fix footer icons

### ISSUE THREAD

**Issue by jenweber** [ember-learn/statusboard#29]
They are broken because they depend on a font that's not installed. Rather than add the font, locate the github, twitter, and google plus SVGs used in Ember-API-docs, save them, and replace the icons in the footer.

This is a temporary fix until the Ember-styleguide component is ready.

**Comment by yohanmishkin** [comment]
I'll see if I can dig SVG's up.

**Comment by yohanmishkin** [comment]
Does this change actually belong in emberjs/website by any chance?

https://github.com/emberjs/website/blob/master/source/_footer.html.erb#L19

**Comment by jenweber** [comment]
@yohanmishkin no, the statusboard app is separate from the website repo. I think the SVGs are [here in the api docs repo](https://github.com/ember-learn/ember-api-docs/tree/master/public/assets/images)

We're moving toward unifying all these into some components from a shared library, called ember-styleguide, so we don't have to copy and paste, but it's not ready yet. Your help fixing the icons for Statusboard in the meantime is very much appreciated! Thank you!

**Comment by jenweber** [comment]
Fixing footer icons turned out to be a hard problem. Still not sure where they are supposed to come from in the glimmer app.

So, we'll swap it out for components, following api docs as a guide:
- https://github.com/ember-learn/ember-api-docs/blob/master/app/components/main-footer.js
- https://github.com/ember-learn/ember-api-docs/blob/master/app/templates/components/main-footer.hbs
- https://github.com/ember-learn/ember-api-docs/blob/master/app/styles/components/_footer.scss
- https://github.com/ember-learn/ember-api-docs/tree/master/public/assets/images
